### PR TITLE
CHANGES

### PR DESCRIPTION
**Pull request by R4d14**
I actually did two thiings, the first one is adding a folder that was never there, and the second one i added a line in README.md file so that you can read me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ This is special file that is intended to show user on how to use the application
 It is very important file that any project should have and should not miss. 
 If it misses we consider the project undone properly.
 >>>>>>> bfddd10 (This is first commit from this project)
+I added this for nothing man
```
Comment (MRadiant) [inline README.md:10]: what is this

File: addedfile [hunk 2]
```diff
@@ -0,0 +1 @@
+This is the new file that i created man, nothing to worry about.
```
Comment (MRadiant) [inline addedfile:1]: I got it

**Review by MRadiant — COMMENTED** [review]
do what is want you to

File: README.md (hunk 1 above)
Comment (R4d14) [reply README.md:10]: This is nothing rather that just a small change

File: README.md [hunk 3]
```diff
@@ -7,3 +7,4 @@ This is special file that is intended to show user on how to use the application
 It is very important file that any project should have and should not miss. 
 If it misses we consider the project undone properly.
 >>>>>>> bfddd10 (This is first commit from this project)
```
Comment (R4d14) [inline README.md:9]: and this also

**Review by MRadiant — APPROVED** [review]
Okay, i got you changes, i am going to merge them

**Pull request merged**